### PR TITLE
Fix typos and some minor editing.

### PR DESCRIPTION
**Pull request by silee2**
Please review these guidelines to help with the review process:
- [ ] Have you provided a meaningful PR description?
- [ ] Have you added a test, a reproducer, or a reference to an issue with a reproducer?
- [ ] Have you tested your changes locally for CPU and GPU devices?
- [ ] Have you made sure that new changes do not introduce compiler warnings?
- [ ] If this PR is a work in progress, are you filing the PR as a draft?
- [ ] Have you organized your commits logically and ensured each can be built by itself?

### CODE REVIEW

**Review by chencha3 — COMMENTED** [review]
but also --> and also ?

**Review by chencha3 — COMMENTED** [review]
it seems .pre-commit-config.yaml is no longer under ./scripts, but under the root (./)

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Intel® Extension for MLIR
-Intel® Extension for MLIR (IMEX) is collection of MLIR dialects and passes from Intel for improving upstream MLIR.
+Intel® Extension for MLIR (IMEX) is a collection of MLIR dialects and passes from Intel for improving upstream MLIR.
 Current extensions covers.
 * Dialects and passes needed to lower and execute Linalg dialect on Intel GPU.
```
[the remark this answers is not in the document]
Comment (Jianhui-Li) [reply README.md:4]: MLIR entry dialect (linalg, CFG, and etc).

**Review by Jianhui-Li — APPROVED** [review]

**Comment by silee2** [comment]
> it seems .pre-commit-config.yaml is no longer under ./scripts, but under the root (./)

Thanks. Corrected.

**Comment by silee2** [comment]
> but also --> and also ?

Which line?

**Review by nbpatel — APPROVED** [review]

**Review by drprajap — APPROVED** [review]

**Pull request merged**